### PR TITLE
commented props part 1

### PR DESCRIPTION
**Pull request by xritter1**
Jira Ticket: [PXP-xxxx](https://occ-data.atlassian.net/browse/COV-544)

### New Features
 - Added comments to properties in observation.yaml, subject.yaml, demographic.yaml, follow_up.yaml, imaging_file.yaml, sample.yaml, summary_location.yaml, statistical_summary_report.yaml, virus_sequence.yaml, and virus_genome.yaml with regards to original datasets. 
- Deleted the property 'bytes' in virus_sequence.yaml (no data migration necessary as there is no data stored for this property)

### CODE REVIEW

File: gdcdictionary/schemas/demographic.yaml [hunk 1]
```diff
@@ -254,7 +254,8 @@ properties:
     description: >
       Province or state
     type: string
-    
+
+# This prop originated from the DS4C dataset
```
Comment (gkuffel) [inline gdcdictionary/schemas/demographic.yaml:258]: Sorry to be a stickler but can we change all of these to:
"Prop originated from [project_name]"

**Review by gkuffel — CHANGES_REQUESTED** [review]

File: gdcdictionary/schemas/demographic.yaml (hunk 1 above)
Comment (xritter1) [reply gdcdictionary/schemas/demographic.yaml:258]: So like this:

Prop originated from DS4C

?

**Review by wangfan860 — DISMISSED** [review]
LGTM! Thank you for adding the annotation `Prop originated from XXX project`

File: gdcdictionary/schemas/observation.yaml [hunk 2]
```diff
@@ -260,6 +262,7 @@ properties:
       - "False"
       - "Unknown"
 
+# Prop originated from coxray project
```
Comment (gkuffel) [inline gdcdictionary/schemas/observation.yaml:265]: Change to COXRAY for consistency.

File: gdcdictionary/schemas/observation.yaml [hunk 3]
```diff
@@ -268,6 +271,7 @@ properties:
...
```
Comment (gkuffel) [inline gdcdictionary/schemas/observation.yaml:274]: same

File: gdcdictionary/schemas/observation.yaml [hunk 4]
```diff
@@ -673,6 +684,7 @@ properties:
       - In recovery
       - Recovered
 
+# Prop originated from nCoDSFSI project
```
Comment (gkuffel) [inline gdcdictionary/schemas/observation.yaml:687]: Project name doesn't exist.

File: gdcdictionary/schemas/virus_genome.yaml [hunk 5]
```diff
@@ -87,11 +87,14 @@ properties:
...
       - GenBank
       - NCBI
+
+  # Prop originated from Sars-cov2 project
   accession:
     description: >
       The accession id of the virus genome.
     type: string
 
+# Prop originated from COXRAY project
```
Comment (gkuffel) [inline gdcdictionary/schemas/virus_genome.yaml:97]: There is no accession_date in the COXRAY project. I believe this is a GDC legacy prop.

**Review by gkuffel — CHANGES_REQUESTED** [review]
Address the few changes and then we can merge.

**Review by hugheslaron — APPROVED** [review]

**Review by gkuffel — APPROVED** [review]
Nice work!

**Review by yilinxu — APPROVED** [review]

**Pull request merged**